test/tools/CMakeLists.txt

```diff
@@ -41,4 +42,5 @@ add_executable(isoltest
 	../libyul/YulOptimizerTestCommon.cpp
 	../libyul/YulInterpreterTest.cpp
 )
-target_link_libraries(isoltest PRIVATE evmc libsolc solidity yulInterpreter evmasm Boost::boost Boost::program_options Boost::unit_test_framework)
+target_compile_definitions(isoltest PRIVATE ISOLTEST)
+target_link_libraries(isoltest PRIVATE evmc libsolc solidity yulInterpreter evmasm Boost::boost Boost::program_options Boost::unit_test_framework Threads::Threads)
```

Comment:
It's an implicit dependency of ``boost::process``, in particular ``child.detach();`` in ``StackLayoutGeneratorTest.cpp``

Reply:
Only breaks the static build - for all others I guess ``Boost::boost`` already pulls it in, but fails to do so for static builds. It's the main reason why I added the ``-DISOLTEST`` define, s.t. this is only really required for the interactive runs.

Reply:
Oh, thanks for mentioning `child.detach()`. I just did not get to that part yet. So basically you need it because you run graphviz as a child process.

It opens a window like you have shown on the call and keeps isoltest hanging in the background until you close it? Or does it render to file?

Reply:
It passes the graph as ``stdin`` to whatever you specify as environment variables, which can do whatever you want with it.
The intended purpose is to have it open a window with the rendered graph, though, as in setting the environment variables to ``dot -Tx11:cairo`` or ``xdot -`` or something like that.